### PR TITLE
Fix: not have empty block statements (ESLint 'no-empty' rule)

### PR DESCRIPTION
**Pull request by ruinername**

### CODE REVIEW

**Comment by ruinername** [comment]
I have one question about rev-manifest. Should it be .json instead .js?

File: root/static/scripts/guess-case/MB/GuessCase/Handler/Track.js [hunk 1]
```diff
@@ -91,10 +91,10 @@ MB.GuessCase.Handler.Track = function (gc) {
...
      */
     self.doWord = function () {
-        if (self.doIgnoreWords()) {
-        } else if (self.doFeaturingArtistStyle()) {
-        } else if (gc.mode.doWord()) {
-        } else {
+        if (self.doIgnoreWords());
+        else if (self.doFeaturingArtistStyle());
+        else if (gc.mode.doWord());
+        else {
```
Comment (mwiencek) [inline root/static/scripts/guess-case/MB/GuessCase/Handler/Track.js]: Here you can combine all three conditions into the first `if`, or assign the result to a `handled` variable like above and do `if (!handled)` for the rest of the matching below.

File: root/static/scripts/guess-case/MB/GuessCase/Handler/Track.js [hunk 2]
```diff
@@ -91,10 +91,7 @@ MB.GuessCase.Handler.Track = function (gc) {
      * - Handles FeaturingArtistStyle
      */
     self.doWord = function () {
-        if (self.doIgnoreWords()) {
-        } else if (self.doFeaturingArtistStyle()) {
-        } else if (gc.mode.doWord()) {
-        } else {
+        if (!self.doIgnoreWords() && !self.doFeaturingArtistStyle() && !gc.mode.doWord()) {
```
Comment (reosarevok) [inline root/static/scripts/guess-case/MB/GuessCase/Handler/Track.js]: Nitpick: this line is too long, it should be split :)

**Comment by mwiencek** [comment]
Thank you!

**Pull request merged**